src/main/java/com/google/sps/lib/ErrorResponse.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.lib;
+
+/** 
+   * This class represents an Error Response.
+   */  
+  public class ErrorResponse {
+    private final Boolean error;
+    private final String error_reason;
```

Comment:
Please read the java style guide. This is not how we name variables in Java.

```suggestion
    private final String errorReason;
```